### PR TITLE
OGM-1430 Support scripting stored procedures for infinispan remote

### PR DESCRIPTION
**Pull request by ntviet18**

### CODE REVIEW

File: core/src/main/java/org/hibernate/ogm/util/impl/Log.java [hunk 1]
```diff
@@ -6,15 +6,9 @@
  */
 package org.hibernate.ogm.util.impl;
 
-import static org.jboss.logging.Logger.Level.DEBUG;
-import static org.jboss.logging.Logger.Level.ERROR;
```
Comment (schernolyas) [inline core/src/main/java/org/hibernate/ogm/util/impl/Log.java]: restore formatting please

**Review by schernolyas — CHANGES_REQUESTED** [review]

File: infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.RESULT_SET_PROC_TITLE_PARAM;
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.UNIQUE_VALUE_PROC_PARAM;
+
+/**
+ * Testing call of stored procedures
+ *
+ * @author The Viet Nguyen &amp;ntviet18@gmail.com&amp;
+ */
+@TestForIssue(jiraKey = { "OGM-1430" })
+public class InfinispanRemoteNamedParametersStoredProcedureScriptTest extends OgmJpaTestCase {
```
Comment (schernolyas) [inline infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java]: Please, use test from 'core' module. All initializations must me in 'prepareDatabase' method

File: infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	}
+
+	@After
+	public void tearDown() {
+		em.close();
+	}
+
+	@Test
+	public void testSingleResultDynamicCall() throws Exception {
+		StoredProcedureQuery storedProcedureQuery = em.createStoredProcedureQuery( "simpleValueProcedure.js" );
```
Comment (schernolyas) [inline infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java]: I think you can check name in cache before

**Review by schernolyas — CHANGES_REQUESTED** [review]

File: infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java (hunk 2 above)
Comment (ntviet18) [reply infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java]: Thank you, I forgot to removed skipped tests. They are enabled and passing now.

**Review by schernolyas — APPROVED** [review]

**Pull request merged**